index.html

```diff
@@ -0,0 +1,321 @@
+              <a href="#"
+                ><img
+                  src="./images/linkedin.png"
+                  alt="linkedin-icon"
+                  class="linkedin"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img
+                  src="./images/medium.png"
+                  alt="medium-icon"
+                  class="medium"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img
+                  src="./images/github.png"
+                  alt="github-icon"
+                  class="github"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img
+                  src="./images/angellist.png"
+                  alt="angellist-icon"
+                  class="angellist"
+              /></a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </div>
+    </div>
+    <!--html of the middle section-->
+    <section class="middle-section">
+      <div class="card1">
+        <div class="snapshoot">
+          <img src="./images/screenshot1.png" alt="screenshot" class="responsive"/>
+        </div>
+        <div class="word-title">
+          <div class="w-title">
+            <h2 class="name">Thomas</h2>
+            <ul class="canop">
+              <li>CANOPY</li>
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <div class="p-text">
+            <p class="word">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+          </div>
+          <div>
+            <ul class="languages">
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+          </div>
+          <img src="./images/see-project.png" alt="button" class="tap">
+        </div>
+      </div>
+      <div class="card2">
+        <div class="snapt">
+          <img src="./images/snapshot4.png" alt="snapshot" class="responsive"/>
+        </div>
+        <div class="titol">
+          <div class="w-title">
+            <h2 class="name">Multi-Post Stories</h2>
+            <ul class="canop">
+              <li>FACEBOOK</li>
+              <li>Full Stack Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <div class="p-text">
+            <p class="word">
+              Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.
+            </p>
+          </div>
+          <div>
+            <ul class="languages-2">
+              <li>html</li>
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+          </div>
+          <img src="./images/see-project.png" alt="button" class="blind">
+        </div>
+      </div>
```

Comment:
- [ ] You have used an image in place of a `<button>` tag for the "See Project" button. Please remove the image and use a button tag instead. Do this for all your images.
- [ ] Also kindly implement the button default, hover and active states as shown in the Figma design below 👇 
![image](https://user-images.githubusercontent.com/46839548/166105675-ca7e6c0d-54bf-4cb3-ad34-5f073ab046d1.png)